amplify/backend/function/getMembersForColony/src/index.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @type {import('aws-lambda').APIGatewayProxyHandler}
+ */
+
+const { Decimal } = require('decimal.js');
+const { constants, utils } = require('ethers');
+const { getColonyNetworkClient, Network } = require('@colony/colony-js');
+const {
+  providers,
+  utils: { Logger },
+} = require('ethers');
+
+Logger.setLogLevel(Logger.levels.ERROR);
+
+const { calculatePercentageReputation } = require('./reputation');
+const { graphqlRequest } = require('./utils');
+const { getUser, getWatchersInColony } = require('./graphql');
+
+/*
+ * @TODO These values need to be imported properly, and differentiate based on environment
+ */
+const API_KEY = 'da2-fakeApiId123456';
+const GRAPHQL_URI = 'http://localhost:20002/graphql';
+
+const ALL_DOMAIN_ID = 0;
+const ROOT_DOMAIN_ID = 1; // this used to be exported from @colony/colony-js but isn't anymore
```

Comment:
We are/will be using these 2 values quite often. Can we have a constants file for all lambda functions like we have for the FE?

Reply:
We're looking into that with Lambda Layers, but for now everything needs to be contained.